api/src/main/kotlin/edu/wgu/osmt/security/AuthHelperService.kt

```diff
@@ -0,0 +1,36 @@
+package edu.wgu.osmt.security
+
+import edu.wgu.osmt.config.UNAUTHENTICATED_USERNAME
+import edu.wgu.osmt.db.PublishStatus
+import edu.wgu.osmt.db.PublishStatus.Archived
+import edu.wgu.osmt.db.PublishStatus.Unarchived
+import org.springframework.stereotype.Service
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.security.oauth2.core.user.OAuth2User
+import org.springframework.security.oauth2.jwt.Jwt
+
+
+@Service
+class AuthHelperService {
```

Comment:
Is this meant as a strategy pattern replacement for auth, or something else?

Reply:
It's just a helper class for OAuth related stuff.

Reply:
Ya, I get that.  My question was more about the intention and form.  How is the code meant to be replaced or overridden from opensource by WGU code?

Reply:
@hikr17 This class won't be overridden in WGU. It's just a class to stash a few methods. they are under unit test.

The SecurityConfig will be overridden, but this is different.